devicedevex_dockerfiles/public/iot-device-cube-1.0.3/azure-c-sdk-public-preview-cross-toolchain-arm32/aarch64-linux-custom.cmake

```diff
@@ -0,0 +1,6 @@
+set(VCPKG_TARGET_ARCHITECTURE arm64)
```

Comment:
Why need the customized CMake settings for arm32?

Reply:
No need of this file. Accidentally copied the useless file under this directory. Will remove it.